main.js

```diff
@@ -0,0 +1,97 @@
+//google maps API key AIzaSyCAAdQ7E1zXvg05JhSDuKCj0NVU89JpmpI
+
+const ApiKey = "b8f04aa7ef4156ea1c923e161e7dcda6";
+var userInput;
+var days = ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday']
+
+getWeather = function() {
+  var current;
+  var fiveDay = [{},{},{},{},{}];
+
+  var refresh = function () {
+    $.getJSON('http://api.openweathermap.org/data/2.5/weather?' + userInput + '&units=imperial&appid=' + ApiKey, function(data) {
+    current = {};
+    current.temp = Math.round(data.main.temp) + '°';
+    current.city = data.name;
+    current.conditions = data.weather[0].description;
+    current.icon = 'http://openweathermap.org/img/w/' + data.weather[0].icon + '.png';
+    displayCurrent();
+    getFiveDay();
+    });
+  };
+
+  var displayCurrent = function () {
+    $('.display').empty();
+    var source = $("#weather-template").html();
+    var template = Handlebars.compile(source);
+    var newHTML = template(current);
+    $('.display').append(newHTML);
+    defaultCity();
```

Comment:
Why invoke this here? I would think this function is "render" function and should do just that